integration_tests/ci/sample.profiles.yml

```diff
@@ -1,13 +1,13 @@
 
-# HEY! This file is used in the Google Ads integrations tests with CircleCI.
+# HEY! This file is used in the dbt package integrations tests with CircleCI.
```

Comment:
Change to Buildkite? Or maybe it'll get batch updated later? 😄

Reply:
Also looks as if `.github/pull_request_template.md` wasn't changed.

Reply:
q1: will be batched later
q2: good catch, updating